getting_started/intel_debug/suggestions.rst

```diff
@@ -27,6 +27,19 @@ enable two of the four speakers present. All of these cases are orthogonal
 to SOF issues in that the SOF driver cannot compensate for codec driver
 problems on its own.
 
+Make sure the ME is enabled
+***************************
+
+If the ME is disabled by the OEM or the user, firmware authentication
+will fail. In case of boot errors, the user should try the following
+command
+
+.. code-block::
+
+   ls /dev/mei*/
+
+If no devices are reported, then the user needs to enable the ME.
```

Comment:
As noted by @fredoh9 , maybe mention here that `/dev/mei*/` requires some Kconfig in the first place.

Also mention that firmware authentication itself does NOT require any `/dev/mei*/`, which could be confusing. Calling it a "debug interface" should avoid any confusion even though it's not just a debug interface.